InnerEye/ML/utils/model_util.py

```diff
@@ -92,6 +92,14 @@ def mean_teacher_model(self) -> Optional[DeviceAwareModule]:
             raise ValueError("Mean teacher model has not been created.")
         return self._mean_teacher_model
 
+    @classmethod
```

Comment:
why is this a class method, and not a static method? Should it live inside this class at all?

Reply:
I've made it a static method. The reason for keeping it in the class was just that everything else related to loading and saving checkpoints is in here, do you think it would be better outside?